app/contexts/adminContext.ts

```diff
@@ -1,23 +1,26 @@
-import { createContext } from 'react'
+import { createContext } from 'react';
 import {
+  CategoryItem,
   EmailTemplateItem,
   PublicPage,
   StoreSettings,
   UserPublicInfo,
-} from '~/types'
+} from '~/types';
```

Comment:
can we follow the same convention, not adding the trailing semicolon?

It's configured in the .prettierrc.json file

```json
{
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "semi": false
}
```